src/services/iam/constants.ts

```diff
@@ -0,0 +1,62 @@
+import { EntityType } from './models/entity-type.js';
```

Comment:
do you use anything from this script, beccause I am confused what this thing is doing here